src/components/organisms/ReviewTemplate/ReviewTemplate.tsx

```diff
@@ -1,40 +1,75 @@
 'use client';
 
-import { Details } from 'nhsuk-react-components';
 import { ZodErrorSummary } from '@molecules/ZodErrorSummary/ZodErrorSummary';
 import { NHSNotifyRadioButtonForm } from '@molecules/NHSNotifyRadioButtonForm/NHSNotifyRadioButtonForm';
-import { ReviewTemplateProps } from './review-template.types';
+import { Container, Row, Col } from 'nhsuk-react-components';
+import { TemplateType, TemplateStatus } from '@utils/enum';
+import concatClassNames from '@utils/concat-class-names';
 import styles from './ReviewTemplate.module.scss';
+import { ReviewTemplateProps } from './review-template.types';
+
+const templateTypeDisplayMappings = (type: TemplateType) =>
+  ({
+    [TemplateType.NHS_APP]: 'NHS App',
+    [TemplateType.SMS]: 'Text message (SMS)',
+    [TemplateType.EMAIL]: 'Email',
+    [TemplateType.LETTER]: 'Letter',
+  })[type];
+
+const templateStatustoDisplayMappings = (status: TemplateStatus) =>
+  ({
+    [TemplateStatus.NOT_YET_SUBMITTED]: 'Not yet submitted',
+    [TemplateStatus.SUBMITTED]: 'Submitted',
+  })[status];
 
 export function ReviewTemplate({
   form,
   ...props
 }: Readonly<ReviewTemplateProps>) {
   return (
     <>
+      <div className='notify-confirmation-panel nhsuk-heading-l'>
+        {props.sectionHeading}
+      </div>
       <ZodErrorSummary errorHeading={form.errorHeading} state={form.state} />
       <h1
         data-testid='preview-message__heading'
         className={styles.review__heading}
       >
-        <span
-          data-testid='preview-message__heading-caption'
-          className='nhsuk-caption-l'
-        >
-          {props.sectionHeading}
-        </span>
-        {props.templateName}
+        {props.template.name}
       </h1>
-      <Details data-testid='preview-message-details'>
-        <Details.Summary data-testid='preview-message-details__heading'>
-          {props.details.heading}
-        </Details.Summary>
-        <Details.Text data-testid='preview-message-details__text'>
-          {props.details.text.map((val) => (
-            <p key={val.id}>{val.text}</p>
-          ))}
-        </Details.Text>
-      </Details>
+      <Container
```

Comment:
Not sure if worth doing since it doesn't quite match UCD design. But there is an NHS react component to do something similar https://main--65aa76b29d00a047fe683b95.chromatic.com/?path=/story/content-presentation-summarylist--summary-list-without-actions

Reply:
interesting! the prototype uses rows and columns like this though so i think it's safer to copy what that's doing